src/workos.ts

```diff
@@ -23,6 +24,7 @@ export class WorkOS {
   readonly auditTrail = new AuditTrail(this);
   readonly directorySync = new DirectorySync(this);
   readonly portal = new Portal(this);
+  readonly passwordless = new Passwordless(this);
```

Comment:
Nit: Looks like the rest of this list is alphabetized so it might be nice to bump it up a line.

Reply:
- [ ] fix module ordering